README.md

```diff
@@ -80,6 +80,25 @@ operate. Using the data model, serializers and deserializers:
     }
     ```
 
+### Zigmod
+
+1. [Read this tutorial](https://nektro.github.io/zigmod/tutorial.html) on how to setup a new zigmod project.
+
+    Note: getty is not hosted on aquila.red ([see this comment for details](https://github.com/nektro/zigmod/issues/63#issuecomment-1063205053)).
+
```

Comment:
We can get rid of this step.

In the Gyro section, I just assume readers know how to set up their project, so we can do the same thing here for Zigmod. And the next step you wrote takes care of not being on Aquila since it modifies `zigmod.yml` instead of using `zigmod aq add`. 

```suggestion
```